entity-service-impl/src/main/java/org/hypertrace/entity/query/service/converter/response/getter/MapValueGetter.java

```diff
@@ -56,16 +61,26 @@ public Value getValue(final JsonNode jsonNode) throws ConversionException {
 
       final String key = entry.getKey();
       final JsonNode node = entry.getValue();
-
-      if (!nestedValueGetter.matches(node)) {
+      boolean valueSet = false;
+      if (nestedValueGetter.matches(node)) {
+        final Value value = nestedValueGetter.getValue(node);
+        Object obj = valueOneOfAccessor.access(value, value.getValueType());
+        valueBuilder.putStringMap(key, obj.toString());
+        continue;
+      }
+      for (ValueGetter getter : rootGetters) {
+        if (getter.matches(node)) {
+          final Value value = this.getValue(node);
```

Comment:
Shouldn't this be getter.getValue(node) instead?

Reply:
yes, my mistake